ssf/README.md

```diff
@@ -1,6 +1,6 @@
-# Simple Sensor Format
+# Standard Sensor Format
```

Comment:
Hahahaha, I really really want it to be "Sensor Sensibility Format" but I guess we can rename it again (: